<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WaveScope</class>
 <widget class="QMainWindow" name="WaveScope">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>816</width>
    <height>604</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WaveScope</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="8">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="4">
       <widget class="QLabel" name="lb_stopbit">
        <property name="text">
         <string>停止位</string>
        </property>
       </widget>
      </item>
      <item row="0" column="6">
       <widget class="QLabel" name="lb_showcnt">
        <property name="text">
         <string>样本点数(个)</string>
        </property>
       </widget>
      </item>
      <item row="1" column="6">
       <widget class="QLineEdit" name="le_showcnt">
        <property name="text">
         <string>800</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QComboBox" name="cb_stopbit">
        <item>
         <property name="text">
          <string>1位</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2位</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="cb_bdrate">
        <property name="currentIndex">
         <number>7</number>
        </property>
        <item>
         <property name="text">
          <string>1200</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2400</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>4800</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>9600</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>19200</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>38400</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>57600</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>115200</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QLabel" name="lb_runtFq">
        <property name="text">
         <string>刷新周期(ms)</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QComboBox" name="cb_parity">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <item>
         <property name="text">
          <string>无校验</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>奇校验</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>偶校验</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QComboBox" name="cb_name"/>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="lb_parity">
        <property name="text">
         <string>校验位</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="lb_databit">
        <property name="text">
         <string>数据位</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="lb_name">
        <property name="text">
         <string>串口</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="lb_bdrate">
        <property name="text">
         <string>波特率</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QComboBox" name="cb_databit">
        <property name="currentIndex">
         <number>3</number>
        </property>
        <item>
         <property name="text">
          <string>5位</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>6位</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>7位</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>8位</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="7" rowspan="2">
       <widget class="QPushButton" name="btn_start">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>打开串口</string>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QLineEdit" name="le_runFq">
        <property name="text">
         <string>5</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>816</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>cb_name</tabstop>
  <tabstop>cb_bdrate</tabstop>
  <tabstop>cb_databit</tabstop>
  <tabstop>cb_parity</tabstop>
  <tabstop>cb_stopbit</tabstop>
  <tabstop>le_runFq</tabstop>
  <tabstop>le_showcnt</tabstop>
  <tabstop>btn_start</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
